html/index.html

```diff
@@ -1,26 +1,27 @@
 <!DOCTYPE html>
 <head>
     <title>My Personal Website</title>
-    <style></style>
 </head>
 <body>
     <header>
-        <img src="/media/cc0-images/grapefruit-slice-332-332.jpg">
+        <img alt="Grapefruit slice atop a pile of other slices" src="/media/cc0-images/grapefruit-slice-332-332.jpg">
         <nav class="quicknav">
             <a href="#News">News</a>
             <a href="#contact">Contact</a>
             <a href="#about">About</a>
        </nav>
-        <div class = "searchbar">
-          <input type="text" placeholder="Search..">
-        </div>
+        <form>
+            <li class="button">
+                <button type="submit">Send your message</button>
+            </li>
```

Comment:
Somehow your input element is now missing. Please check again the article I recommended to you in my last code review about the html structure of forms. 🙏  As for the challenge itself, it is not necessary to add a form, you can also remove this part for now and add it later after our session about forms (planned for next week). :)